lib/bible_bot/bible.rb

```diff
@@ -6,7 +6,7 @@ class Bible
         id: 1,
         name: "Genesis",
         abbreviation: "Gen",
-        regex: "^(?:Ge|Gen)(?:esis)?",
+        regex: "(?:Ge|Gen)(?:esis)?",
```

Comment:
This `^` was added so that Genesis wouldn't be selected for Judges. But this creates a separate problem, in that it will only select a Genesis reference if it is at the very beginning of the string.

A better solution is down in `Book.find_by_name`, where I added `\b` to the beginning of the Regex. This ensures it is only selecting for whole words. So `/ge/` will only match if the word starts with `ge`, not if it contains `ge` as is the case for Judges.